src/list.tsx

```diff
@@ -24,6 +24,7 @@ import {getPrefixedDataAttributes} from './utils/dom';
 
```

Comment:
please, can you define some new type tests in `list-tipe-tests.tsx` to cover the cases this PR is fixing?